.github/workflows/ci.yml

```diff
@@ -7,20 +7,45 @@ on:
   push:
     branches:
       - master
+    tags: [ v* ]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '8', '11', '17' ]
-        scala: [ '2.12.15' ]
+        java:
+          - name: Zulu OpenJDK 8
+            version: zulu@1.8
+          - name: Zulu OpenJDK 11
+            version: zulu@1.11
+          - name: Zulu OpenJDK 17
+            version: zulu@1.17
+    name: With ${{ matrix.java.name }}
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK
-      uses: actions/setup-java@v2
-      with:
-        java-version: ${{ matrix.java }}
-        distribution: 'zulu'
-    - name: Run tests
-      run:  sbt ++${{ matrix.scala }} "^ test"
+      - uses: actions/checkout@v2.4.0
+      - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: ${{ matrix.java.version }}
+      - uses: coursier/cache-action@v6
+      - name: Test
+        run: sbt test
+  publish:
+    name: Publish release
+    if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
+    needs: [build]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: zulu@1.8
+      - uses: olafurpg/setup-scala@v13
+      - run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
You will need to set these four secrets

Reply:
@ghostbuster91 sorry for my late response, but a simple question `PGP_SECRET` is my pgp private key and pgp_passphrase is the passphrase for it?

Reply:
well I figured it already out, I also needed to base64 it. i know thats why I created `jib@briefdomain.de`. (well I should've used jib@gccc.de but I did not have email setup for that domain I guess)